### ISSUE TITLE
Feature Improvement - from_dataframe() -> 'TimeseriesDataRaw':

### ISSUE THREAD

**Issue by JotaBlanco** [quixai/quix-streams#53]
I understand that in streaming is quite uncommon to have big chunks (batches) of data been publish, so a long dataframe with lots or rows is unlikely. 
Anyhow, right now the dataframe to TimeseriesDataRaw conversion is done by row and column, which is inefficient. I'll be proposing a new vectorized version to improve speed (specially with big dataframes).

**Comment by JotaBlanco** [comment]
Ok, interesting, I've been playing with a vectorized version of the existing [from_dataframe() ](https://github.com/quixio/quix-streams/blob/main/src/PythonClient/src/quixstreams/models/timeseriesdataraw.py#L134) and this is what I've found:

- My new code is much faster with big dataframes (above ~250 rows) and much slower with smaller dataframes.
![image](https://user-images.githubusercontent.com/50139826/223111271-988c8ecd-26d0-4df4-bcde-0141ce7e9574.png)

- Given that most messages will have a low number of rows (streaming data, not batches of data), I think the current code should stay. 

- I still like to share my vectorized version for future reference, what's the best way? @peter-quix 

- Whilst doing this exercise, I found two bugs that need solving:
    - bug 1: from_dataframe brakes if there are nulls in the time column. I've raised a bug: https://github.com/quixio/quix-streams/issues/54
    - bug 2: from_dataframe brakes if there are complex numbers in the dataframe. I've raised a bug: https://github.com/quixio/quix-streams/issues/55
